src/main/java/com/agilethought/internship/sso/controller/UserController.java

```diff
@@ -0,0 +1,34 @@
+package com.agilethought.internship.sso.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.agilethought.internship.sso.services.UserService;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+@RestController
+@RequestMapping(value = "/api/v1")
+@Api(value = "Software Cost Estimation", tags = "SCE")
+public class UserController {
```

Comment:
We currently have a controller for this API, I think we must use it instead of create new one, just in case that previous controller has same purpose. what do you think?